includes/classes/shopping_cart.php

```diff
@@ -611,19 +611,20 @@ function remove_all() {
   }
   /**
    * Method return a comma separated list of all products in the cart
+   * NOTE: Not used in core ZC, but some plugins make use of it as a helper function
    *
    * @return string
-   * @todo ICW - is this actually used anywhere?
```

Comment:
Even better ... I'll update it to mention shipping modules specifically, for future clarity.